package.json

```diff
@@ -4,16 +4,19 @@
   "description": "hello",
   "main": "index.js",
   "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
```

Comment:
The "start" script is added, but the "nodemon" dependency is only in devDependencies. If you plan to use nodemon for the "start" script in a production environment, you should move it to dependencies. [medium]